glib/src/variant.rs

```diff
@@ -1163,7 +1163,10 @@ impl ToVariant for std::path::PathBuf {
 #[cfg(any(unix, windows))]
```

Comment:
This `cfg`s can go away now. `glib::translate` always provides these helper functions and they're unconditionally used (indirectly) from elsewhere too.